testLibOPHD/testLibOPHD.vcxproj

```diff
@@ -162,7 +162,7 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClCompile Include="main.cpp" />
+    <ClCompile Include="MapOffset.cpp" />

```

Comment:
Just noting that we never renamed this file to `MapOffset.test.cpp`, which would match the convention in other test projects.